src/index.js

```diff
@@ -0,0 +1,89 @@
+
+
+import * as readline from 'node:readline/promises'
+import { arrayToString } from './utils/arrayUtils.js';
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+});
+
+
+function progressGuess(randomNumber, guess, prevGuess) {
+    if (randomNumber > Number(guess)) {
+        console.log('업')
+        console.log('이전 추측:', arrayToString(prevGuess))
+        return
+    }
+
+    if (randomNumber < Number(guess)) {
+        console.log('다운')
+        console.log('이전 추측:', arrayToString(prevGuess))
+        return
+    }
+
+    console.log('정답!')
+    return 'correct'
+}
+
+async function askPlayAgain() {
```

Comment:
`다시 시작한다`라는 의미를 한 단어로 아래와 같이 표현해줘도 괜찮을 것 같네요~
```suggestion
async function askRestart() {
```